Mathlib/Topology/ApproximateUnit.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2024 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+import Mathlib.Topology.Algebra.Monoid
+
+/-! # Approximate units
+
+An *approximate unit* is a filter `l` such that multiplication on the left (or right) by `m : α`
+tends to `𝓝 m` along the filter, and additionally `l ≠ ⊥`.
+
+Examples of approximate units include:
+
+- The trivial approximate unit `pure 1` in a normed ring.
+- `𝓝 1` or `𝓝[≠] 1` in a normed ring (note that the latter is disjoint from `pure 1`).
+- In a C⋆-algebra, the filter generated by the sections `fun a ↦ {x | a ≤ x} ∩ closedBall 0 1`,
```

Comment:
This is still true (and slightly stronger) if you replace `closedBall` by `ball` right? Anyway, we can talk about this in the following PR.